src/components/FaqNl.vue

```diff
@@ -146,6 +146,18 @@
       door je leverancier. Wij ondersteunen je hier graag bij.
     </p>
 
+    <h3 class="display-5">
+      Hoe kan ik OpenRefine gebruiken met het Termennetwerk?
```

Comment:
Assuming that readers already know what OpenRefine is. Alternatively, we could frame this question as: ‘Hoe kan ik mijn data matchen met het Termennetwerk?’

Reply:
I think readers know what OpenRefine is, so the current title sounds good to me :)